packages/typography/src/index.css

```diff
@@ -0,0 +1,65 @@
+:root {
+  /* Agnostic values */
+  --natural-line-height: calc((var(--ascender) - var(--descender)));
+  --line-height-scale: calc(
+    var(--natural-line-height) *
+    1rem /
+    var(--units-per-em)
+  );
+  --ascender-scale: calc(var(--ascender) * 1rem / var(--units-per-em));
+}
+
+/* Reset */
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+}
+
+/* Where the magic happens */
+/* Add or remove selectors as needed. The baseline alignment will apply to the selectors included here */
+h1,
+h2,
+h3,
+h4,
+p {
+  --computed-line-height: var(
+    --line-height,
+    calc(var(--baseline-height) * var(--line-height-multiplier))
+  );
+  line-height: var(--computed-line-height);
+  font-size: var(--font-size);
+  background: #fededebb;
```

Comment:
This is applying a reddish tinted background to all text

![Screenshot 2024-12-20 at 14 57 26](https://github.com/user-attachments/assets/ba798355-5265-4182-8c61-0a58d9d4c8a0)